exchanges/exchanges.go

```diff
@@ -3005,17 +3022,84 @@ func (dcr *DecredDEX) Refresh() {
 		return
 	}
 
+	candlesticks := make(map[candlestickKey]Candlesticks)
+	var change float64
+	var volume, bestVolDur uint64
+	var aDayMS uint64 = 86400 * 1000
+	// Ugh. I need to export the CandleCache.candles.
```

Comment:
Yeah. `CandleCache` stores them internally as `[]dexdb.Candle`, but since the field isn't exported and there's no getter, I'm converting to `msgsjon.WireCandles` just to recreate the `[]dexdb.Candle`.